root/static/scripts/common/components/Annotation.js

```diff
@@ -82,14 +84,18 @@ const Annotation = ({
                 date: formatUserDate($c.user, annotation.creation_date),
                 user: <EditorLink editor={annotation.editor} />,
               })}
+              {' '}
               {numberOfRevisions && numberOfRevisions > 1 ? (
                 <>
-                  {' '}
                   <a href={entityHref(entity, '/annotations')}>
                     {l('View annotation history')}
                   </a>
+                  {' / '}
                 </>
               ) : null}
+              <a href={returnUri($c, entityHref(entity, 'edit_annotation'), 'returnto', $c.req.uri)}>
```

Comment:
Where does it take you without setting `returnto`?

Reply:
Seems to go back to the same spot - should I just remove the returnto then?

Reply:
If the Annotation component is only used on the index pages (which I think is the case), I'd just remove it, since that's where it'll return by default.